content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -472,33 +514,14 @@ installing Kubernetes from scratch. If this is the case, you need to manually
 install the `conntrack` package (e.g. `sudo apt install conntrack` on Ubuntu)
 and then retry.
 
-### Is kube-proxy writing iptables rules?
-
-One of the main responsibilities of `kube-proxy` is to write the `iptables`
-rules which implement `Services`.  Let's check that those rules are getting
-written.
+Kube-proxy can run in one of a few modes.  Youc an see in the log above the
```

Comment:
nit: reword or fix the `Youc an`
nit: possible reword of lines 517-518,
In the log listed above, the line `Using iptables Proxier` indicates that kube-proxy is running in iptables mode.